_posts/2021-05-22-fluxv2-openfaas.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "See how Flux's Git Ops ToolKit can simplify delivery"
+description: "See how Flux can simplify serverless deployments to OpenFaaS"
+date: 2021-05-22    
+image: /images/2021-05-22-flux/earth.jpg
+categories:
+- arkade
+- kubectl
+- gitops
+- openfaas-operator
+- flux
+- CI/CD
+author_staff_member: alistair
+dark_background: true
+
+---
+
+We learn how to manage OpenFaaS and install functions using Flux.
+
+# What is Flux
```

Comment:
The introduction should talk about the original version of flux and what's new in flux v2 / GitOps Toolkit.

Stylised: WeaveWorks